applications/volume_rendering/python/metadata.json

```diff
@@ -13,8 +13,9 @@
 			"1.0": "Initial Release"
 		},
 		"holoscan_sdk": {
-			"minimum_required_version": "2.1.0",
+			"minimum_required_version": "2.0.0",
 			"tested_versions": [
+				"2.0.0",
```

Comment:
1.0.1 tested working by adding `GXF::multimedia` to `operators/volume_renderer/CMakeLists.txt`

```
target_link_libraries(volume_renderer
  PRIVATE
    holoscan::core
    clara::viz::renderer
    clara::viz::core
    clara::viz::core::hw
    GXF::multimedia <==== add this line
)
```

Reply:
Added to the PR (marked you as co-author)